app/src/store/playground/types.ts

```diff
@@ -96,6 +97,8 @@ export type ModelConfig = {
   modelName: string | null;
   endpoint?: string | null;
   apiVersion?: string | null;
+  // TODO(apowell): Should we store this by provider/model? Should it be stored outside of the model config?
+  invocationParameters: Partial<Omit<InvocationParameters, "toolChoice">>;
```

Comment:
I don'tt hink we need to store it by provider / model. Since there is only one provider / model per instance t a time. Is your thought that if someone has temperature = 0.3 then switch to a model with no temperature then back you want to preserve .3 and not go back to 0 or whatever?

Reply:
I was kinda just writing these comments to capture my thoughts as I went along and then I forgot to remove them. I think it makes sense to _not_ store it by provider/model, in agreement with your reasoning.

Reply:
Removed comment